docs/fa/examples.md

```diff
@@ -0,0 +1,97 @@
+# Examples
+
+[FA Class](class-fa.md)  
+[NFA Class](class-nfa.md)  
+[DFA Class](class-dfa.md)  
+[Table of Contents](../README.md)
+
+On this page, we give some short examples with discussion for the finite automata classes and methods.
+
+## Reading basic input
+
+The following code snippet creates a `DFA` and prints whether it accepts or rejects user input.
+
+```python
+from automata.fa.dfa import DFA
+
+# DFA which matches all binary strings ending in an odd number of '1's
+my_dfa = DFA(
+    states={'q0', 'q1', 'q2'},
+    input_symbols={'0', '1'},
+    transitions={
+        'q0': {'0': 'q0', '1': 'q1'},
+        'q1': {'0': 'q0', '1': 'q2'},
+        'q2': {'0': 'q2', '1': 'q1'}
+    },
+    initial_state='q0',
+    final_states={'q1'}
+)
+
+try:
+    while True:
+        if my_dfa.accepts_input(input('Please enter your input: ')):
+            print('Accepted')
+        else:
+            print('Rejected')
+except KeyboardInterrupt:
+    print('')
+```
+
+## Subset for NFAs
+
+The `NFA` does not have a built-in method for checking whether it is a subset of another `NFA`. However, this can be done using existing methods.
+
+```python
+# In the following, we have nfa1 and nfa2 and want to determine whether nfa1 is a subset of nfa2.
+
+# If taking the union of nfa2 with nfa1 is equal to nfa2 again,
+# nfa1 didn't accept any strings that nfa2 did not, so it is a subset.
+if (nfa1 | nfa2) == nfa2:
+    print('nfa1 is a subset of nfa2.')
+else:
+    print('nfa1 is not a subset of nfa2.')
+
+```
+
+## Edit distance automaton
+
+The following example is inspired by [this blog post](http://blog.notdot.net/2010/07/Damn-Cool-Algorithms-Levenshtein-Automata). Essentially, given a set of strings, we want to determine whether 
```

Comment:
@eliotwrobson When the URL is especially long, I tend to prefer reference-style links, per https://daringfireball.net/projects/markdown/syntax#link:

```markdown
The following example is inspired by [this blog post][levelshtein-article].
Essentially, given a set of strings, we want to determine whether 

[levelshtein-article]: http://blog.notdot.net/2010/07/Damn-Cool-Algorithms-Levenshtein-Automata)
```

Also, the last sentence is incomplete ("We want to determine whether ...").

Reply:
Addressed here: https://github.com/caleb531/automata/pull/146/commits/2ce6816e437d8ca17b4e095ab4922e18eb4421f7